### ISSUE TITLE
blockinghandler set order to false is not work

### ISSUE THREAD

**Issue by huangliequan** [eclipse-vertx/vert.x#3971]
### Questions
I set Request to BlockingHandler and Order to False.But the tasks are still executed serially, not in parallel

![image](https://user-images.githubusercontent.com/10766750/122219436-019e1480-cee2-11eb-84d2-ad2ec3ef698b.png)

![image](https://user-images.githubusercontent.com/10766750/122219247-d287a300-cee1-11eb-9652-fcaf9970fbb9.png)

### Version

Which version(s) did you encounter this bug ? 4.1.0

**Comment by tsegismont** [comment]
Closing as duplicate of #3954. This will be fixed in 4.1.1.
